script.js

```diff
@@ -1,6 +1,45 @@
 function runProgram(){
 
     console.log('runProgram');
-    //your code goes here
+    const api_url ="https://zenquotes.io/api/quotes/";
+    getapi(api_url)
+
+    let searchButton = document.querySelector(`#Button`)
+    searchButton.addEventListener('click', findQuote);
+}
+document.addEventListener('DOMContentLoaded', runProgram);
+let data;
+
+async function getapi(url)
+{
+  const response = await fetch(url);
+  data = await response.json();
+
+
+  const arr = ["hi", "hello"];
+  console.log(arr[1])
+
+  console.log(data[2]);
```

Comment:
Kathryn, it would have been better to remove these practice/example lines of code before turning in your code